app/main/views.py

```diff
@@ -35,12 +35,14 @@ def index():
 
 
 @main.route('/index.html', methods=['GET'])
+@main.route('/status', methods=['GET'])
 def status():
-    return 200
+    return '', 200
 
 
 @main.route('/contact', methods=['GET', 'POST'])
-def contact():
+@main.route('/technical-support', methods=['GET', 'POST'])
```

Comment:
Can we make this `/technical_support`

Reply:
https://stackoverflow.com/questions/5362954/why-use-instead-off-in-url

Reply:
Makes sense. Lets leave it as `/technical-support`.